src/Email.tsx

```diff
@@ -5,6 +5,7 @@ import { Banner } from "./components/Banner";
 import { Collections } from "./components/Collections";
 import { FilmToday } from "./fronts/film-today/FilmToday";
 import { MediaBriefing } from "./fronts/media-briefing/MediaBriefing";
+import { SportAus } from "./fronts/sport-aus/SportAus";
```

Comment:
I just noticed the front ID is actually `sport-au` so we should probably change our files and folders to match that - so adopt terminology 'Sport AU' or 'sport-au` throughout instead of 'Sport AUS' or `sport-aus`.

Reply:
Updated